### PR TITLE
Update New-CsTeamsMeetingPolicy.md

### PR DESCRIPTION
**Pull request by vini-gupta**
Adding new meetingPolicy setting AllowNetworkConfigurationSettingsLookup to help determines whether network configuration setting lookup can be made for not enterprise voice enabled users. It will be used for enabling network roaming policy

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9

File: skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md [hunk 1]
```diff
@@ -970,6 +971,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -AllowNetworkConfigurationSettingsLookup
+Determines whether network configuration setting lookup can be made for users who are not enterprise voice enabled, it will be used for enabling network roaming policy
```
Comment (JohanFreelancer9) [inline skype/skype-ps/skype/New-CsTeamsMeetingPolicy.md:975]:
```suggestion
Determines whether network configuration setting lookups can be made by users who are not Enterprise Voice enabled. It is used to enable Network Roaming policies.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @vini-gupta, kindly commit the suggested change if you agree with it. Many thanks.

**Comment by yogkumgit** [comment]
@vini-gupta kindly implement the suggested changes. Thanks

**Comment by yogkumgit** [comment]
@dariomws @tseward @get-itips

**Review by dariomws — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tseward

**Comment by yogkumgit** [comment]
@tseward please review

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Pull request merged**